frontend/src/containers/LibraryPage/LibraryPage.js

```diff
@@ -213,6 +148,7 @@ class LibraryPage extends React.Component {
                 onSelect={handleSelect}
                 videoCards={videoCards[uploadDate]}
                 selectedCards={selectedCards}
+                onVideoCardLoad={this.fetchThumbnail}
```

Comment:
is this prop used in `VideoCardsByDate` component?

Reply:
Nope, forgot about it